src/analysis/convex/basic.lean

```diff
@@ -1628,7 +1722,7 @@ end
 
 /-- All values of a function `f ∈ std_simplex ι` belong to `[0, 1]`. -/
 lemma mem_Icc_of_mem_std_simplex (hf : f ∈ std_simplex ι) (x) :
-  f x ∈ I :=
+  f x ∈ Icc (0 : ℝ) 1 :=
```

Comment:
`I` is defined as localised notation in [`topology.unit_interval`](https://leanprover-community.github.io/mathlib_docs/topology/unit_interval.html), but it probably doesn't really matter here.